<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QVinACE</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,100,3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="m_start_button">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_stop_button">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="m_reset_button">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="m_top_extra_widget" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="m_monitor_frame">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background:black</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,5">
        <item>
         <widget class="QFrame" name="m_bottom_extra_frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Computer">
    <property name="title">
     <string>&amp;Computer</string>
    </property>
    <addaction name="action_Start"/>
    <addaction name="actionSto_p"/>
    <addaction name="separator"/>
    <addaction name="action_Reset"/>
    <addaction name="action_IRQ"/>
    <addaction name="action_NMI"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Computer"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Start">
   <property name="text">
    <string>&amp;Start</string>
   </property>
  </action>
  <action name="actionSto_p">
   <property name="text">
    <string>Sto&amp;p</string>
   </property>
  </action>
  <action name="action_Reset">
   <property name="text">
    <string>&amp;Reset</string>
   </property>
  </action>
  <action name="action_NMI">
   <property name="text">
    <string>&amp;NMI</string>
   </property>
  </action>
  <action name="action_IRQ">
   <property name="text">
    <string>&amp;IRQ</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
